packages/core/src/components/StepList/Step.jsx

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import StepLink from './StepLink';
+import SubStep from './SubStep';
+import classNames from 'classnames';
+import { stepShape } from './StepList';
+
+export const Step = ({ step, ...props }) => {
+  const start = step.isNextStep;
+  const resume = step.started && !step.completed;
+  const className = classNames('ds-c-step', {
+    'ds-c-step--current': start || resume,
+    'ds-c-step--completed': step.completed
+  });
+  return (
+    <li className={className}>
+      <h2 className="ds-c-step__header ds-h3 ds-u-display--flex ds-u-justify-content--between">
```

Comment:
https://cms.invisionapp.com/share/STEPOUPYB#/screens is the visual I'm comparing to, and it would seem H2 is the right tag, when thinking about it from a screen reader user who navigates via headers. `Get information to figure out your premium tax credit` feels like an H1, because it states the page's intent, and these steps would be the next logical hierarchical stopping point.